server/chat-plugins/rock-paper-scissors.ts

```diff
@@ -373,6 +360,11 @@ export const commands: ChatCommands = {
 			game.unpause(user);
 		},
 
+		start() {
+			const game = getGame(this);
+			game.start();
+		},
```

Comment:
Why do you need a start command?

Reply:
Actually, shouldn't be necessary. Will remove.